utils/documentation_tests.txt

```diff
@@ -38,6 +38,7 @@ src/transformers/models/speech_encoder_decoder/modeling_speech_encoder_decoder.p
 src/transformers/models/speech_to_text/modeling_speech_to_text.py
 src/transformers/models/speech_to_text_2/modeling_speech_to_text_2.py
 src/transformers/models/swin/modeling_swin.py
+src/transformers/models/trocr/modeling_trocr.py
```

Comment:
nice ❤️!

Are you able to run the doctest locally and get it pass?

Reply:
Yep, runs fine locally! I ran the test again after addressing your comments